src/main/scala/ch/epfl/bluebrain/nexus/kg/resources/Resources.scala

```diff
@@ -220,4 +279,6 @@ object Resources {
                      schemaImports: Set[ResourceV],
                      dataImports: Set[ResourceV],
                      data: Graph): EitherT[F, Rejection, Unit] = ???
+
+  private case class Resolved(schema: ResourceV, dataImports: Set[ResourceV], schemaImports: Set[ResourceV])
```

Comment:
I would suggest renaming to `SchemaContext` because it's quite specific to schemas.